src/App.js

```diff
@@ -1,12 +1,19 @@
 import React, { useEffect } from 'react';
 import { ThemeProvider, CSSReset } from '@chakra-ui/core';
 import { useGapiDispatch } from './hooks/GAPI';
+
+import { useBooksDetailsDispatch, useBooksDispatch } from './hooks/books';
 import Router from './router';
 
 function App() {
+    const { loadPublishersAndSeries } = useBooksDetailsDispatch();
+    const { loadDesignations } = useBooksDispatch();
     const { loadGapi } = useGapiDispatch();
     useEffect(() => {
         loadGapi();
+        loadPublishersAndSeries();
+        loadDesignations();
+        // eslint-disable-next-line
     }, []);
```

Comment:
The stuff in `useEffect` will, because of the `[]` only be run for the very first time, right?

Reply:
yes , that is correct ,it just needs to load the load the google API and initialize it , and get the list of publishers and categories from our server